Packs/NutanixHypervisor/Classifiers/classifier-mapper-incoming-Nutanix_Hypervisor.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
where is the classifier?

Reply:
Arseny said to hard-code the type in the code. That's why there is no classifier